app/main.py

```diff
@@ -49,9 +56,17 @@ def __init__(self):
             'amqp://{}:{}@{}:{}/%2f'.format(
                 self._default_user,
                 self._default_pass,
-                self._rabbit_hosts[1],
+                self._rabbit_hosts[0],
                 self._rabbit_port) + settings.HEARTBEAT_INTERVAL,
         ]
+        
+        self._aws_url = [
+            'amqp://{}:{}@{}:{}/%2f'.format(
+                self._aws_user,
+                self._aws_pass,
+                self._aws_rabbit_corp_host[0],
+                self._corp_rabbit_port)
+        ]
 
         self.publisher = QueuePublisher(
             self._urls,
```

Comment:
The Queue publishers and Quarantine publishers can be changed to point to AWS

  self.publisher = QueuePublisher(	        
 		self._aws_url,
        settings.COLLECT_QUEUE,	            
        )